string name                         # object's name
std_msgs/ColorRGBA color            # new color
--- 
